AI_CHATBOT_DOCUMENTATION.md

```
@@ -0,0 +1,248 @@
+
+3. **Start Development Server**
+
+```bash
+npm run dev
+# or
+bun dev
+```
+
+4. **Test the Chatbot**
+
+- Open http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)